addons/crm/security/ir.model.access.csv

```diff
@@ -22,4 +22,4 @@ access_calendar_event_manager,calendar.event.manager,calendar.model_calendar_eve
 access_calendar_event,calendar.event,calendar.model_calendar_event,sales_team.group_sale_salesman,1,1,1,0
 access_calendar_event_type_sale_manager,calendar.event.type.manager,calendar.model_calendar_event_type,sales_team.group_sale_manager,1,1,1,0
 access_calendar_event_type_sale_user,calendar.event.type.user,calendar.model_calendar_event_type,base.group_user,1,0,0,0
-access_calendar_event_type_sale_user,calendar.event.type.salesman,calendar.model_calendar_event_type,sales_team.group_sale_salesman,1,0,0,0
+access_calendar_event_type_sale_salesman,calendar.event.type.salesman,calendar.model_calendar_event_type,sales_team.group_sale_salesman,1,0,0,0

```

Comment:
It's duplicated from https://github.com/odoo/odoo/blob/1d34e269dac2e0cafd85ad91a6240569b5b7282c/addons/crm/security/ir.model.access.csv#L24